cmd/sealer/cmd/alpha/alpha.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// NewCmdAlpha returns "sealer alpha" command.
+func NewCmdAlpha() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "alpha",
+		Short: "Sealer experimental sub-commands",
```

Comment:
Please add a long field to add more description of `sealer alpha` command. This is very useful for end users. Thanks. And do not make sealer uppercase. sealer is a reserved word, then it has no uppercase mode, just like etcd. sealer and etcd both have a lowercase first letter.

Reply:
`alpha` command has sub command which has their own long description. i am ok with adding long description on `alpha` command, but seems like there is very little to describe.

Reply:
😄， we should explain to help end-users to understand what is alpha command for. Please take the following paragraph into consideration.

```
alpha command of sealer is used to provide functionality incubation from immature to mature. Each function will experience a growing procedure. Alpha command policy calls on end users to experience alpha functionality as early as possible, and actively feedback the experience results to sealer community, and finally cooperate to promote function from incubation to graduation. Please file an issue at https://github.com/sealerio/sealer/issues when you have any feedback on alpha commands.
```